scripts/test-uma-ODRL-policy.ts

```diff
@@ -4,45 +4,134 @@
-    console.log("expecting only policy 1: ", res.body);
+    let response = await fetch(endpoint(), { headers: { 'Authorization': client('a') } })
+    console.log("expecting policy 1, 1a and <urn:uuid:95efe0e8-4fb7-496d-8f3c-4d78c97829bc>: \n", await response.text())
+
+    response = await fetch(endpoint(), { headers: { 'Authorization': client('b') } })
+    console.log("expecting policy 2, 2a and <urn:uuid:95efe0e8-4fb7-496d-8f3c-4d78c97829bc>: \n", await response.text())
+
+    response = await fetch(endpoint(), {});
+    console.log(`expecting 4xx error code (no authorization header provided): ${response.status}`)
 }
 
-async function main() {
-    console.log(`Primitive unit test to check policy access based on the client\n`);
+async function getOnePolicy() {
+    console.log("Simple test for the GET One Policy endpoint");
+
+    const encoded = encodeURIComponent(policyId);
 
-    await testGetAllPolicies();
+    let response = await fetch(endpoint(`/${encoded}`), { headers: { 'Authorization': client('a') } });
+    console.log(`expecting to return relevent information about ${policyId}`, await response.text());
+
+    response = await fetch(endpoint(`/${badPolicyId}`), { headers: { 'Authorization': client('b') } });
+    console.log(`expecting 4xx error code since the policy ID is not valid: ${response.status}`)
+
+    response = await fetch(endpoint(`/${encoded}`), { headers: { 'Authorization': client('c') } });
+    console.log(`expecting 4xx error code since the client is not authorized to access the policy: ${response.status}`)
+}
 
+async function postPolicy() {
+    console.log("Simple test for the POST policy endpoint");
+
+    let response = await fetch(endpoint(), { method: 'POST', headers: { 'Authorization': client('a'), 'Content-Type': 'text/turtle' }, body: Buffer.from(policyB, 'utf-8') });
+    console.log(`expecting a negative response since assigner != client: status code ${response.status}\nmessage: ${await response.text()}`);
+
+    response = await fetch(endpoint(), { method: 'POST', headers: { 'Authorization': client('a'), 'Content-Type': 'text/turtle' }, body: Buffer.from(policyA, 'utf-8') });
+    console.log(`expecting a positive response: status code ${response.status}, ${await response.text()}`);
+
+    response = await fetch(endpoint(), { method: 'POST', headers: { 'Authorization': client('b'), 'Content-Type': 'text/turtle' }, body: Buffer.from(policyB, 'utf-8') });
+    console.log(`expecting a positive response: status code ${response.status}, ${await response.text()}`);
+
+    response = await fetch(endpoint(), { method: 'POST', headers: { 'Authorization': client('c'), 'Content-Type': 'text/turtle' }, body: Buffer.from(policyC, 'utf-8') });
+    console.log(`expecting a positive response: status code ${response.status}, ${await response.text()}`);
+}
+async function main() {
+    console.log("Testing all implemented Policy Endpoints:\n\n\n");
+    await postPolicy();
```

Comment:
So this both tests and loads policies in the server. Could you document which policies are expected to be in there?
Because otherwise, the next steps will not work